src/QuadForm/Lattices.jl

```diff
@@ -966,6 +966,46 @@ function restrict_scalars(L::AbsLat)
   return ZLat(Vabs, Mabs)
 end
 
+@doc Markdown.doc"""
+    restrict_scalars_with_map(L) -> 
+       Tuple{ZLat, Hecke.SpaceRes}}
+
+Given a hermitian lattice `L`, return the $\mathbb Z$-lattice obtained by 
```

Comment:
```suggestion
Given a lattice `L`, return the $\mathbb Z$-lattice obtained by 
```
this is on me, I forgot to make the distinction for hermitian lattices